app/src/main/java/org/fossasia/susi/ai/adapters/recycleradapters/ChatFeedRecyclerAdapter.java

```diff
@@ -259,6 +264,8 @@ public int getItemViewType(int position) {
         if (item.getId() == -404) return DOTS;
         else if (item.getId() == -405) return NULL_HOLDER;
         else if (item.isDate()) return DATE_VIEW;
+        else if (item.getContent() != null && (item.getContent().endsWith(".jpg")||(item.getContent().endsWith(".png"))))
```

Comment:
Actually the thing is the link coming from the server are from pixabay.com and they all end with .jpg